.github/workflows/pr-name.yml

```
@@ -0,0 +1,19 @@
+name: Pull Request Name
+on:
+  pull_request:
+    branches:
+      - next
+
+jobs:
+  pull_request_name:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Obtain PR name
+        run: echo "PR_TITLE=${{ github.event.pull_request.title }}" >> $GITHUB_ENV
```

actionlint (1.7.7)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)